index.html

```diff
@@ -1408,21 +1408,21 @@
   </div>
   <!-- ends Play VGA SMPTE bars -->
 
-  <!-- Broken DV -->
-  <span data-toggle="modal" data-target="#broken_dv"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Make a broken DV file from a perfectly good one">Broken DV</button></span>
-  <div id="broken_dv" class="modal fade" tabindex="-1" role="dialog">
+  <!-- Broken File -->
+  <span data-toggle="modal" data-target="#broken_file"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Make a broken file out of a perfectly good one">Broken file</button></span>
+  <div id="broken_file" class="modal fade" tabindex="-1" role="dialog">
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
-          <h3>Makes a broken DV test file</h3>
-          <p>Modifies an existing, functioning DV file and intentionally breaks it for testing purposes.</p>
-          <p><code>ffmpeg -i <i>input_file</i>.dv -bsf noise -c copy <i>output_file</i>.dv</code></p>
+          <h3>Makes a broken video (or audio) test file</h3>
```

Comment:
Is `a broken audio and/or video test file` an idea? I mean: both on the same level.